toolchain/lower/handle.cpp

```diff
@@ -310,4 +310,7 @@ auto HandleInst(FunctionContext& context, SemIR::InstId inst_id,
                        llvm::PointerType::get(context.llvm_context(), 0)));
 }
 
+auto HandleInst(FunctionContext& /*context*/, SemIR::InstId /*inst_id*/,
+                SemIR::Vtable /*inst*/) -> void {}
+
```

Comment:
```suggestion
```

Given the lowering settings on `Vtable`, can this be removed?

Reply:
Ah, yep - removed 35b27ffe77b4bd2a6f4591bc57b064bd3e4920dc (oh, didn't realize you'd made the suggested edit too... or I would've used that).